src/Extensions/Features/src/FeatureCollection.cs

```diff
@@ -161,6 +161,6 @@ private sealed class FeatureCollectionDebugView(FeatureCollection features)
         private readonly FeatureCollection _features = features;
 
         [DebuggerBrowsable(DebuggerBrowsableState.RootHidden)]
-        public KeyValuePair<string, object>[] Items => _features.Select(pair => new KeyValuePair<string, object>(pair.Key.FullName ?? string.Empty, pair.Value)).ToArray();
+        public DictionaryItemDebugView<Type, object>[] Items => _features.Select(pair => new DictionaryItemDebugView<Type, object>(pair)).ToArray();
```

Comment:
Does `KeyValuePair` need to be replaced here because the `KeyValuePair` doesn't implement the DebugProxyView in the runtime?

Reply:
Yes. The magic is the debug display attribute on this type:

```cs
[DebuggerDisplay("{Value}", Name = "[{Key}]")]
```